Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -262,6 +262,14 @@ configuration:
   section: Collect
   advanced: true
   required: false
+- name: extensive_logging
```

Comment:
looks like display field is missing.
Should be "Extensive Logging"

Reply:
added